week-2/D-predicates/exercise.js

```diff
@@ -7,12 +7,12 @@
 
 // Finish the predicate function to test if the passed number is negative (less than zero)
 function isNegative(number) {
-
+  return typeof number > 0;
 }
 
 // Finish the predicate function to test if the passed number is between 0 and 10
 function isBetweenZeroAnd10(number) {
-
+  return typeof number > 0 < 10;
```

Comment:
that's not correct - what happens when number is `-20`? And what happens when the number is `40`? Does it give you sensible answers? The clue is in `typeof` - why did you use it?